src/lib/components/workspace/team-invite/TeamInvite.svelte

```diff
@@ -216,39 +232,49 @@
 
 <div class="mt-4">
   <p class="role-title mb-1">Role<span class="asterik">*</span></p>
-  <SelectRoleDropdown
-    isError={roleError && selectedRole === "select"}
-    id={"invite-member-workspace"}
+  <Dropdown
+    dropDownType={{ type: "text", title:selectedRole ?selectedRole : "" }}
+    dropdownId="invite-team"
     data={[
       {
         name: "Select",
         id: "select",
         description: "Select role",
-        color: "whiteColor",
+       dynamicClasses: "text-whiteColor",
+        hide: true,
       },
       {
         name: "Admin",
-        id: TeamRole.TEAM_ADMIN,
+        id: WorkspaceRole.WORKSPACE_ADMIN,
         description:
           "Add & edit resources within a workspace,add & remove members to workspace",
-        color: "whiteColor",
+         dynamicClasses: "text-whiteColor",
       },
       {
         name: "Editor",
         id: WorkspaceRole.WORKSPACE_EDITOR,
         description: "Add & edit resources within a workspace",
-        color: "whiteColor",
+         dynamicClasses: "text-whiteColor",
       },
       {
         name: "Viewer",
         id: WorkspaceRole.WORKSPACE_VIEWER,
         description: "View Resources within a workspace.",
-        color: "whiteColor",
+        dynamicClasses: "text-whiteColor",
       },
     ]}
```

Comment:
move this to a const file and then import it here

Reply:
These are not same. here there are workspace level role and in other file , it is a team level role also dynamicClasses will consist bootstrap classes  which are same for now , but may contain more classes in future. also each option can have more properties in future,so not really common.